code/game/objects/structures/tables_racks.dm

```diff
@@ -125,7 +125,7 @@
 		if(!G.confirm())
 			return 0
 		G.affecting.loc = src.loc
-		G.affecting.Weaken(2)
+		G.affecting.Weaken(5)

```

Comment:
That's a pretty big buff; make it 3 or 4 and I'll be more willing to merge
Baby steps